ibm_mq/datadog_checks/ibm_mq/config.py

```diff
@@ -91,9 +105,11 @@ def __init__(self, instance):
         custom_tags = instance.get('tags', [])  # type: List[str]
         self.tags_no_channel = [
             "queue_manager:{}".format(self.queue_manager_name),
-            "mq_host:{}".format(self.host),  # 'host' is reserved and 'mq_host' is used instead
-            "port:{}".format(self.port),
+            "mq_host:{}".format(host),  # 'host' is reserved and 'mq_host' is used instead
+            "port:{}".format(port),
+            "connection_name:{}".format(self.connection_name),
```

Comment:
if a user sets a `connection_name`, would the `mq_host` and `port` tags have the value `None`?

Reply:
Yes, that's right. I can maybe not send them if `mq_host` and `port` are none, what do you think?

Reply:
Yes, I think not including the `mq_host` and `port` tags if their values are none might be better. It might cause confusion if customers get a `None` value tag and may not be helpful.

Reply:
Fixed.